nodebuilder/p2p/reachability.go

```diff
@@ -23,6 +23,11 @@ func reachabilityCheck(ctx context.Context, host HostBase) {
 		for {
 			select {
 			case <-ticker.C:
+				if autoNAT == nil {
```

Comment:
should it be before `for` ???

Reply:
this variable is not gonna change anyway, so we're going loop for no reason and spamming with this message (not sure is this the expected behaviour or not).